tools/logger/convert.c

```diff
@@ -68,7 +68,7 @@ static void log_err(FILE *out_fd, const char *fmt, ...)
 	if (buff) {
 		vsprintf(buff, fmt, args);
 		fprintf(stderr, "%s", buff);
```

Comment:
```suggestion
		fprintf(stderr, "Error: %s", buff);
```
And remove the Error in the original message, will make it easy to keep align for error message.

Reply:
Yes, I also thought about that. I'm sitting to work